### PR TITLE
GH-110383: Improve Tutorial for Input Ouput

### PR DESCRIPTION
**Pull request by blaisep**
In order to get[ GH-118970](https://github.com/python/cpython/pull/118970)  from @uatach merged quickly, I updated the PR with some formatting changes suggested by @encukou 

----
📚 Documentation preview 📚: https://cpython-previews--119230.org.readthedocs.build/

* Issue: gh-110383

### CODE REVIEW

File: Doc/tutorial/inputoutput.rst [hunk 1]
```diff
@@ -197,7 +204,12 @@ notation. ::
    Jack: 4098; Sjoerd: 4127; Dcab: 8637678
 
 This is particularly useful in combination with the built-in function
-:func:`vars`, which returns a dictionary containing all local variables.
+:func:`vars`, which returns a dictionary containing all local variables. ::
```
Comment (encukou) [inline Doc/tutorial/inputoutput.rst]:
```suggestion
:func:`vars`, which returns a dictionary containing all local variables::
```

File: Doc/tutorial/inputoutput.rst [hunk 2]
```diff
@@ -37,16 +37,23 @@ printing space-separated values. There are several ways to format output.
...
+  block there are two examples of how to format variables:
+
 
   ::
 
      >>> yes_votes = 42_572_654
-     >>> no_votes = 43_132_495
-     >>> percentage = yes_votes / (yes_votes + no_votes)
+     >>> total_votes = 43_132_495
+     >>> percentage = yes_votes / (yes_votes + total_votes)
```
Comment (encukou) [inline Doc/tutorial/inputoutput.rst]: Total votes allows simplifying the expression (to put more focus on the actual formatting):
```suggestion
     >>> total_votes = 85_705_149
     >>> percentage = yes_votes / total_votes
```

**Comment by blaisep** [comment]
@encukou Nice adjustment, thank you.

**Review by encukou — APPROVED** [review]
Thank you @uatach and @blaisep! Let's get it in!

**Pull request merged**